_posts/en/newsletters/2022-01-26-newsletter.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Bitcoin Optech Newsletter #184'
+permalink: /en/newsletters/2022/01/26/
+name: 2022-01-26-newsletter
+slug: 2022-01-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to extend PSBTS with fields
+for spending outputs constructed using a pay-to-contact protocol and
+includes our regular sections with summaries of top posts from the
+Bitcoin Stack Exchange and notable changes to popular Bitcoin
+infrastructure projects.
+
+## News
+
+- **PSBT extension for P2C fields:** Maxim Orlovsky [proposed][orlovsky
+  p2c] a new BIP to add optional fields to [PSBTs][topic psbt] for
+  spending from an output created using a [Pay-to-Contract][topic p2c]
+  (P2C) protocol, as previously mentioned in [Newsletter #37][news37
+  psbt p2c].  P2C allows a spender and a receiver to agree on the text
+  of a contract (or anything else) and then create a public key that
+  commits to that text.  The spender can then later demonstrate that the
+  payment committed to that text and that it would've been
+  computationally infeasible for that commitment to have been made
+  without the cooperation of the receiver.  In short, the spender can
+  prove to a court or the public what they paid for.
+
+    However, in order for the receiver to subsequently construct a
+    signature spending their received funds, they need the hash of the
+    contract in addition to the key they used (that key usually being
+    part of an [HD keychain][topic bip32]).  Orlovsky's proposal allows
+    that hash to be added to a PSBT so that a signing wallet or hardware
+    device can produce a valid signature.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to convert a taproot address into a native segwit address?]({{bse}}111440)
+  After an exchange changed a user's segwit v1 P2TR taproot withdrawal address
+  into a segwit v0 native segwit P2WSH address due to lack of taproot support,
```

Comment:
Both P2TR and P2WSH are native segwit addresses. Also, "segwit v0 native segwit" feels a bit off. Perhaps this would be better:

```suggestion
  After an exchange changed a user's P2TR (native segwit v1) taproot withdrawal address
  into a P2WSH (native segwit v0) address due to lack of taproot support,
```